src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: $Smoothly;
```

Comment:
it's a good practice to add transition popery, because now transition is applied for all properties